tools/monitor.js

```diff
@@ -0,0 +1,87 @@
+import Arweave from 'arweave';
+import { connect, createDataItemSigner } from '@permaweb/aoconnect';
+import {
+  ANTRegistry,
+  AOProcess,
+  ANT_REGISTRY_ID,
+  ARIO,
+  ARIO_MAINNET_PROCESS_ID,
+} from '@ar.io/sdk';
+import { pLimit } from 'plimit-lit';
+
+const arweave = Arweave.init({
+  host: 'arweave.net',
+  port: 443,
+  protocol: 'https',
+});
+
+async function main() {
+  const jwk = await arweave.wallets.generate();
+
+  const ao = connect({
+    CU_URL: 'https://cu.ardrive.io',
+    GRAPHQL_URL: 'https://arweave.net/graphql',
+  });
+  const ario = ARIO.init({
+    process: new AOProcess({
+      processId: ARIO_MAINNET_PROCESS_ID,
```

Comment:
make this an env var

Reply:
[1cc9df8](https://github.com/ar-io/ar-io-ant-registry-process/pull/16/commits/1cc9df8dd28863d53b0fe033e1f34211cabcd15f)